frontend/dockerfile/docs/reference.md

```diff
@@ -2224,6 +2224,12 @@ RUN echo "I'm building for $TARGETPLATFORM"
 | `BUILDKIT_SYNTAX`                     | String | Set frontend image                                                                                                                                                                                             |
 | `SOURCE_DATE_EPOCH`                   | Int    | Set the UNIX timestamp for created image and layers. More info from [reproducible builds](https://reproducible-builds.org/docs/source-date-epoch/). Supported since Dockerfile 1.5, BuildKit 0.11              |
 
+> **Warning**
+>
+> `BUILDKIT_INLINE_BUILDINFO_ATTRS` build arg is deprecated and will be removed
+> in a future release. Use [provenance attestations](https://docs.docker.com/build/attestations/slsa-provenance/)
```

Comment:
This is indeed always the tricky bit: "future release" isn't great (but sometimes needed if we don't know _yet_ when we'll remove), but _ideally_ we would either have "remove in version X" or "removed in next release" (if it's the next release).